executor/inspection_result.go

```diff
@@ -287,9 +291,102 @@ func (configInspection) inspectCheckConfig(_ context.Context, sctx sessionctx.Co
 			})
 		}
 	}
+	results = append(results, c.checkTiKVBlockCacheSizeConfig(ctx, sctx, filter)...)
 	return results
 }
 
+func (c configInspection) checkTiKVBlockCacheSizeConfig(ctx context.Context, sctx sessionctx.Context, filter inspectionFilter) []inspectionResult {
+	item := "storage.block-cache.capacity"
+	if !filter.enable(item) {
+		return nil
+	}
+	sql := "select instance,value from information_schema.cluster_config where type='tikv' and `key` = 'storage.block-cache.capacity'"
+	rows, _, err := sctx.(sqlexec.RestrictedSQLExecutor).ExecRestrictedSQLWithContext(ctx, sql)
+	if err != nil {
+		sctx.GetSessionVars().StmtCtx.AppendWarning(fmt.Errorf("check configuration in reason failed: %v", err))
+	}
+	extractIP := func(addr string) string {
+		if idx := strings.Index(addr, ":"); idx > -1 {
+			return addr[0:idx]
+		}
+		return addr
+	}
+
+	ipToBlockSize := make(map[string]uint64)
+	ipToCount := make(map[string]int)
+	for _, row := range rows {
+		ip := extractIP(row.GetString(0))
+		size, err := c.convertReadableSizeToByteSize(row.GetString(1))
```

Comment:
Can we add a hint `output_raw_format` for `"select instance,value from information_schema.cluster_config where type='tikv' and `key` = 'storage.block-cache.capacity'"` and make the `row.GetString(1)` a number, instead of string like `20KiB`. So we could remove the function `convertReadableSizeToByteSize()`

Reply:
em...Sound great, but hard to this, `20Kib` value was return by TiKV HTTP API, Use hint `output_raw_format ` still need to do a special handle in other places, and use function `convertReadableSizeToByteSize`.

Any great succession for a better place to handle `output_raw_format ` hint?